test/multiverse/suites/hybrid_agent/parsing_helpers.rb

```diff
@@ -157,7 +157,14 @@ def verify_span_output(output)
       end
 
       expected['attributes']&.each do |expected_key, expected_value|
-        assert_equal expected_value, actual[1] & [expected_key],
+        # 0 is intrinsic attributes, 1 is custom attributes, 2 is agent attributes
```

Comment:
Alternatively, we could flatten the hash or something instead, which might make more sense